src/main/java/com/es/phoneshop/enums/Order.java

```diff
@@ -0,0 +1,16 @@
+package com.es.phoneshop.enums;
+
+public enum Order {
```

Comment:
Please rename this enum to avoid ambiguity (order is the thing user places after adding products to a cart)